chart/kubeapps/templates/kubeappsapis/deployment.yaml

```diff
@@ -85,6 +85,21 @@ spec:
           env:
             - name: PORT
               value: {{ .Values.kubeappsapis.containerPort | quote }}
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: ASSET_SYNCER_DB_URL
+              value: {{ template "kubeapps.postgresql.fullname" . }}-headless:{{ default "5432" .Values.postgresql.service.port }}
+            - name: ASSET_SYNCER_DB_NAME
+              value: {{ .Values.postgresql.postgresqlDatabase }}
+            - name: ASSET_SYNCER_DB_USERNAME
+              value: postgres
+            - name: ASSET_SYNCER_DB_USERPASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: postgresql-password
+                  name: {{ include "kubeapps.postgresql.secretName" . }}
```

Comment:
We should somehow indicate that this config should be temporary only (ie. we don't want to be providing such config for plugins) until we can switch to the caching provided to plugins that will eventually come out of #2946 . You've said something similar when pulling in the utils.go temporarily below.